src/Datadog.Trace.ClrProfiler.Native/dd_profiler_constants.h

```diff
@@ -51,7 +51,6 @@ namespace trace {
 
   WSTRING skip_assemblies[]{
       "mscorlib"_W,
-      "netstandard"_W,
```

Comment:
I don't think you need to remove this line and look at the netstandard assembly, since the database calls won't be originating from netstandard. Other libraries will just look for `DbCommand` or `IDbCommand` inside of `netstandard.dll`

Reply:
You're right! Fixed in 964aa86e